src/refactorings/convert-switch-to-if-else/convert-switch-to-if-else.test.ts

```diff
@@ -165,11 +165,34 @@ default:
 }`,
         expected: `if (name === "Jane") {
   return sayHelloToJane();
-} else if (name === "John") {
+}
+
+if (name === "John") {
   return sayHelloToJohn();
-} else {
+}
+
+return sayHello();`
+      },
+      {
+        description: "with return statements and fallthrough",
+        code: `switch (name) {
+case "Jane":
+  return sayHelloToJane();
+case "John":
+case "Johnny":
+  return sayHelloToJohn();
+default:
   return sayHello();
-}`
+}`,
+        expected: `if (name === "Jane") {
+  return sayHelloToJane();
+}
+
+if (name === "John" || name === "Johnny") {
+  return sayHelloToJohn();
+}
+
+return sayHello();`
```

Comment:
👍 Tests look great!

What if you have a mix of return statements and non-return?

Reply:
I see there's `allEndWithReturn` in the code so I guess it will behave as expected.

Can you still add a test to illustrate?

Reply:
I'll merge and do it, so I can release a new version today ;)

Thanks for your hard work 🙏